tests/integration/conftest.py

```diff
@@ -64,4 +74,57 @@ def time_series(request, cognite_client):
     create_subscription_in_cdf(timeseries_set, sub_name, cognite_client)
     yield timeseries_set
     remove_time_series_data(timeseries_set, cognite_client)
-    cognite_client.time_series.subscriptions.delete(sub_name)
\ No newline at end of file
+    cognite_client.time_series.subscriptions.delete(sub_name)
+
+@pytest.fixture(scope="session")
+def test_space(cognite_client: CogniteClient):
+    space = cognite_client.data_modeling.spaces.retrieve(SPACE_ID)
+    if space is not None:
+        yield space
+    else:
+        new_space = SpaceApply(SPACE_ID, name="Integration Test Space", description="The space used for integration tests.")
+        yield cognite_client.data_modeling.spaces.apply(new_space)
+    cognite_client.data_modeling.spaces.delete(spaces=[SPACE_ID])
+    
+
+@pytest.fixture(scope="session")
+def test_dml():
+    return (RESOURCES / "movie_model.graphql").read_text()
+
+@pytest.fixture(scope="function")
+def test_model(cognite_client: CogniteClient, test_space: Space, test_dml: str):
+    movie_id = DataModelId(space=test_space.space, external_id="Movie", version="1")
+    created = cognite_client.data_modeling.graphql.apply_dml(
+        id=movie_id, dml=test_dml, name="Movie Model", description="The Movie Model used in Integration Tests"
+    )
+    models = cognite_client.data_modeling.data_models.retrieve(created.as_id(), inline_views=True)
+    yield models.latest_version()
+    cognite_client.data_modeling.data_models.delete(ids=[movie_id])
+    views = models.data[0].views
+    for view in views: # Views and containers need to be deleted so the space can be deleted
+        cognite_client.data_modeling.views.delete((SPACE_ID, view.external_id, view.version))
+        cognite_client.data_modeling.containers.delete((SPACE_ID, view.external_id))
+
+@pytest.fixture(scope="function")
+def edge_table_name(azure_credential):
+    edge_table_name = os.environ["LAKEHOUSE_ABFSS_PREFIX"] + "/Tables/" +  SPACE_ID + "_edges"
```

Comment:
Maybe a helper method to get the table name rather than have duplicate code in edge_table_name & view_tables

Reply:
Added